File: CS/FilterDuplicateRecords_Converter/RoundConverter.cs
using System;
using System.Globalization;
using System.Windows.Data;

namespace FilterDuplicateRecords_Converter
{
    internal class RoundConverter : IValueConverter
    {
        private const int decimals = 2;

        public object Convert(object value, Type targetType, object parameter, CultureInfo culture) {
            if (value is decimal number) {
                return Math.Round(number, decimals);
            }

            throw new ArgumentException();
        }

        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture) {
            return value;
        }
    }
}

File: VB/FilterDuplicateRecords_Converter/RoundConverter.vb
Imports System
Imports System.Globalization
Imports System.Windows.Data

Namespace FilterDuplicateRecords_Converter

    Friend Class RoundConverter
        Implements IValueConverter

        Private Const decimals As Integer = 2

        Public Function Convert(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As CultureInfo) As Object Implements IValueConverter.Convert
            If TypeOf value Is Decimal Then
                Return Math.Round(CDec(value), decimals)
            End If

            Throw New ArgumentException()
        End Function

        Public Function ConvertBack(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As CultureInfo) As Object Implements IValueConverter.ConvertBack
            Return value
        End Function
    End Class
End Namespace
